packages/router-generator/src/config.ts

```diff
@@ -10,6 +10,7 @@ export const configSchema = z.object({
   generatedRouteTree: z.string().optional().default('./src/routeTree.gen.ts'),
   quoteStyle: z.enum(['single', 'double']).optional().default('single'),
   semicolons: z.boolean().optional().default(false),
+  tabWidth: z.number().optional().default(2),
```

Comment:
Please remove the `tabWidth` configuration as we've decided to stop adding in code-styling-related (prettier) configurations.

I haven't gotten around to it yet, but we need to update the documentation to explicitly ask the user to *ignore* the generated route-tree file from both their linter (eslint, biome, etc) and their formatted (prettier,...).

The rationale behind this is quite simple. This file is something controlled by the router-generator and shouldn't be touched by the linters or formatters.

Reply:
got it 🙏
I'm already ignored the route tree file in my formatter.